types/src/interpolation.c

```diff
@@ -79,11 +82,20 @@ static rage_Error validate_time_series(
     return RAGE_OK;
 }
 
-static rage_FrameSeries * as_frameseries(
-        rage_TupleDef const * const tuple_def, rage_TimeSeries const * points,
-        uint32_t const sample_rate, uint8_t const awaiting_completion) {
-    rage_FrameSeries * fs = malloc(sizeof(rage_FrameSeries));
-    fs->c = rage_countdown_new(awaiting_completion);
+static void frameseries_free(
```

Comment:
Is this really `frameseries_destroy` because it doesn't free the memory for the actual `rage_FrameSeries` argument?